inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/gather/gather_kernel_ref.cpp

```diff
@@ -8,6 +8,10 @@
 #include <vector>
 
 namespace kernel_selector {
+static size_t GetGatherBatchDim(const gather_params& params) {
```

Comment:
What's the point of this method? It doesn't do any processing, and doesn't improve readability of the code.